exercises/hello-world/HINTS.md

```diff
@@ -14,12 +14,11 @@ but it helps to understand what a test looks like, and what
 it is doing.
 
 Open up the test file, hello-world.spec.js.
-It has three tests defined in it.
 
 This is the first test:
```

Comment:
Perhaps this should be changed to say 'There is one test:' or something of that nature?